### PR TITLE
Adding the hostnames to dhcp for etcd HA, updated README for etcd

### LINKED ISSUE #19: Multi-ETCD node deployment, networking issue
All machines are using 10.0.2.15 as an IP, causing issues.

See gist: https://gist.github.com/stephenrlouie/763c08eee8a02d5920ac0902d66557f4

### PR DESCRIPTION
**Pull request by stephenrlouie**
- Changes to close #19 
- Confirmed and updated [current image] (https://atlas.hashicorp.com/stephenrlouie/boxes/gemini_master/)

### CODE REVIEW

File: vagrant/README.md [hunk 1]
```diff
@@ -53,6 +53,16 @@
  kube_download_url_base: http://192.168.2.2/downloads/bins/kubernetes/v{{ kube_version }}/
  pypy_base_url: http://192.168.2.2/downloads/bins/pypy/v{{ pypy_version }}/
  ```
+   - Configures the right adapters for flannel and etcd
+
+ ```
+ #The adapter to use for flannel
```
Comment (danehans) [inline vagrant/README.md]: We should not hard code the adapter name as they are different across platforms, releases, etc.. How is contrib vagrant handling interfaces and lets follow a similar pattern.

Comment (stephenrlouie) [inline vagrant/README.md]: This is how upstream vagrant does it

https://github.com/kubernetes/contrib/commit/730c9724e8fcc31d8dbb68c6534c540a3a7b04fb

Comment (danehans) [inline vagrant/README.md]: OK, then we should match. When I deploy using contrib vagrant my vbox shows eth0, eth1, etc.. Does gemini vagrant show enp0s3 b/c we're using a newer version of cent7? If not, what is causing the discrepancy between the two?

File: vagrant/provision_files/dhcpd.conf [hunk 2]
```diff
@@ -26,46 +26,57 @@ subnet 192.168.2.0 netmask 255.255.255.0 {
...
     }
     host core_os_node10 {
     hardware ethernet 00:00:00:00:00:0a;
             fixed-address 192.168.2.12;
+            option host-name "node10";
     }
     host core_os_node11 {
     hardware ethernet 00:00:00:00:00:0b;
             fixed-address 192.168.2.13;
+            option host-name "node11";
```
Comment (danehans) [inline vagrant/provision_files/dhcpd.conf]: If it doesn't currently exist, can you please add to the vagrant doc's stating the max node count?

File: vagrant/README.md (hunk 1 above)
Comment (stephenrlouie) [inline vagrant/README.md]: #40 - fixed

File: vagrant/provision_files/dhcpd.conf (hunk 2 above)
Comment (stephenrlouie) [inline vagrant/provision_files/dhcpd.conf]: https://github.com/gemini-project/gemini/pull/39/files#diff-a33810049da88ccb0b4c6fb4d2b3c757R77

**Pull request merged**